modules/graphql_core/graphql_core.info.yml

```diff
@@ -1,7 +1,6 @@
 name: GraphQL Core
 type: module
 description: 'Provides type system plugins and derivers on behalf of core modules.'
-package: GraphQL
-core: 8.x
+package: Testing
 dependencies:
-  - graphql
\ No newline at end of file
+  - graphql
```

Comment:
I know this is small, but we should be using the new format for dependencies, `graphql:graphql`